File: Movil/ProyectoABCJava/app/src/main/java/com/example/proyectoabc/service/ApiAutenticacion.java
package com.example.proyectoabc.service;

import com.example.proyectoabc.main.Usuario;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.Headers;
import retrofit2.http.POST;

public interface ApiAutenticacion {
    @FormUrlEncoded
    @Headers("Accept: application/json")
    @POST("login")
    Call<Usuario>LOGIN_CALL(
            @Field("usuario") String usuario,
            @Field("contrasena") String contrasena
    );
}

File: Movil/ProyectoABCKotlin/app/src/main/java/com/example/proyectoabckotlin/service/ApiAutenticacion.kt
package com.example.proyectoabckotlin.service;

import com.example.proyectoabckotlin.pojo.Usuario
import retrofit2.Call;
import retrofit2.http.Body
import retrofit2.http.POST;

public interface ApiAutenticacion {
    /*@GET("/login/{id}")
    fun login(@Path("id") id: String): Call<Usuario>

    @GET("/login?usuario={usuario}")
    fun login(@Query("usuario") usuario: String): Call<Usuario>*/

    @POST("/users/auth")
    fun login(@Body usuario: Usuario): Call<Usuario>
}

